packages/react-components/src/components/EndCallButton.tsx

```diff
@@ -63,10 +62,16 @@ export const EndCallButton = (props: EndCallButtonProps): JSX.Element => {
     styles ?? {}
   );
 
+  const onHangUp = (): void => {
```

Comment:
What is the point of this function?

Reply:
This function is there so we can call onHangup() with no params then pass it to onClick
We cannot pass onHangup directly to onClick anymore since the parent component is passing in onHangup(foreveryone: boolean) into endcallbutton